packages/backend/hardware-sec.go

```diff
@@ -0,0 +1,26 @@
+package main
+import (
+    "strings"
+    "os/exec"
+    "context"
+    "google.golang.org/grpc/codes"
+    "google.golang.org/grpc/status"
+    pb "github.com/canonical/desktop-security-center/packages/proto"
+    "google.golang.org/protobuf/types/known/emptypb"
+)
+
+func (s *HardwareServer) IsSecureBootEnabled(ctx context.Context, p *emptypb.Empty) (*pb.IsSecureBootEnabledResponse, error) {
+    cmd := exec.Command("mokutil", "--sb-state")
+    out, err := cmd.Output()
+    if err != nil {
+        return &pb.IsSecureBootEnabledResponse{ Enabled: true }, status.Errorf(codes.NotFound, "couldn't execute mokutil")
```

Comment:
Must this really be this painful? I was expecting I could just `return true, status.Errorf(...)`

Reply:
Usually in Go, you will either return a response or an error, and `nil` the other. Because this is an error case, you can do this:
```
if err != nil {
    return nil, status.Errorf(codes.NotFound, "couldn't execute mokutil")
}
``` 
You can do the same for that final else clause that has an error.

On a side note, you could probably swap this from a series of if else statements to a [switch statement](https://go.dev/tour/flowcontrol/9), as all the cases seem distinct to me.

Reply:
I see, I'll update it to return nil in those cases then, thanks!

But for the other cases I was just annoyed by the verbosity of `&pb.IsSecureBootEnabledResponse{ Enabled: true }` instead of just `true`, but well that was just style in the end of the day.